### DISCUSSION TITLE
Is there a reason the Arrow costs so much, relative to other similar ships?

### DISCUSSION THREAD

**Discussion by brendanjones** [endless-sky/endless-sky#9995] [Q&A] [answered]
The Arrow costs 2.9 million, with 2000 shields and 400 hull. This doesn't seem right when you compare it to similar ships.

Especially compared to the Scout, which costs literally half the Arrow but has mostly the same specs (and is far superior in cargo and fuel). Or the Headhunter, which costs 300K less but is a far superior ship in all but speed.

I'd like to suggest reducing the Arrow's cost by about 500,000, but wanted to check first if I'm missing something. The fact that it comes with atomic engines and luxury accommodation still doesn't add up to the 2.9 million cost, IMO.

<img width="275" alt="Screenshot 2024-04-15 at 14 12 35" src="https://github.com/endless-sky/endless-sky/assets/16049594/d2701b78-676c-46c5-9cb7-b725c7a781f4">
<img width="282" alt="Screenshot 2024-04-15 at 14 13 32" src="https://github.com/endless-sky/endless-sky/assets/16049594/bff36b33-f205-482f-93da-15bab9af78d5">
<img width="282" alt="Screenshot 2024-04-15 at 14 12 39" src="https://github.com/endless-sky/endless-sky/assets/16049594/012d316c-0cff-4f17-bfe0-4ff372845171">
<img width="279" alt="Screenshot 2024-04-15 at 14 12 52" src="https://github.com/endless-sky/endless-sky/assets/16049594/9c42c93d-cf39-4585-b5fc-38542d6567bf">
<img width="280" alt="Screenshot 2024-04-15 at 14 13 02" src="https://github.com/endless-sky/endless-sky/assets/16049594/6db24e6e-851f-4945-8fab-f2040732cd5a">
<img width="269" alt="Screenshot 2024-04-15 at 14 20 03" src="https://github.com/endless-sky/endless-sky/assets/16049594/876e2e79-ed7a-4fca-a9d0-f27ebe823910">

**Comment by Hecter94** [comment, marked as answer]
Effectively, it's a lore reason, not a mechanical reason. 
The arrow's description is:

> "The Arrow is a light transport, designed for bringing a small number of very rich passengers - Syndicate CEOs, for example - from one planet to another with speed and style.

It's overpriced because it's catering to the ultra-rich and likely comes with all of the extra amenities and luxuries that the super-rich expect, which doesn't translate to any better actual in-game performance. 

For context, I imagine the interior of a clipper probably looks something like:
![image](https://github.com/endless-sky/endless-sky/assets/15916854/74722994-1b8c-4633-9188-d52d052e78d8)

Whereas the Arrow's interior probably looks something like:
![image](https://github.com/endless-sky/endless-sky/assets/15916854/e046610d-24b5-4bf7-9084-18b554755d50)

**Reply by brendanjones** [reply]
Yeah this is how I perceived it, also. But at a practical level, isn't the pre-installed luxury accommodation providing that luxury? Okay, maybe the whole ship is fancy, not just the accommodation bit, so yes it can still be more expensive than equivalent ships. 

But if it was dropped 500K in price it'd still be more expensive than all equivalent ships (and slightly cheaper than the Headhunter, which it _should_ be cheaper than), and it'd just maybe be considered by players.

I mean, have you ever bought an Arrow? I certainly never have. If I had 2.9 million cash I'd buy an Argosy.

**Reply by Zitchas** [reply]
Yes, I have bought an arrow. Not recently, but now and then for dinner pilots it feels like the right choice to me.

Not every pilot is min maxing economic potential.

**Reply by brendanjones** [reply]
Dinner Pilot?

**Reply by Hecter94** [reply]
> Yeah this is how I perceived it, also. But at a practical level, isn't the pre-installed luxury accommodation providing that luxury? Okay, maybe the whole ship is fancy, not just the accommodation bit, so yes it can still be more expensive than equivalent ships.

I consider the luxury accommodations outfit to be different from an actual "luxury" ship. Installing that outfit doesn't mean the entire interior living space of your ship gets completely redone to be fancier. I consider it to be something more equivalent to a high-end lounge area—a central area with luxury accommodations where rich passengers can go to be treated to fine food, drinks, and fancy surroundings. This somewhat explains why it affects your entire ship. 

Of course, the actual outfit sprite basically disproves that theory. It heavily implies it's just a single fancy room with a single bed that all of your rich passengers share, but I choose to ignore that because it's nonsensical. 
![image](https://github.com/endless-sky/endless-sky/assets/15916854/983f0bcf-6cf4-4bd9-be20-07f12b3ce0d4)


> But if it was dropped 500K in price it'd still be more expensive than all equivalent ships (and slightly cheaper than the Headhunter, which it _should_ be cheaper than), and it'd just maybe be considered by players.

It comes down to the fact that not everything in the world of Endless Sky needs to exist solely for the player. Some things can just be world-building. The arrow isn't a useful ship in pure gameplay terms, but it's valuable to others in Endless Sky who aren't the player. Thus, it has a valid reason to exist and improves the believability and immersion quality of the world we're building.  

~~and it's not like you're particularly starved for options in human space anyways~~

**Comment by Zitchas** [comment]
It is worth noting that the hull cost of the Arrow and the Scout are not terribly far apart. The Arrow is definitely more expensive, but fairly reasonable. (numbers may not be exact, since I'm pulling from Delta, not vanilla, so take with a measure of salt)

Arrow is 1.2M
Scout is 0.85M

The vast majority of the difference in price tag is due to outfits. The Scout comes by default with Ion engines, for instance, and the Arrow with atomics. That difference *alone* accounts for almost half a million in price difference. 

The presence of a Luxury accommodations instead of a ramscoop is another 190k increase in the Arrow's performance. 

Overall, the Arrow starts with 1.7M worth of outfits, compared to the Scout's 428k.  So that's roughly an extra million on the price tag that doesn't have much to do with the hull at all, and is instead just because of what it comes with.

**Comment by xX-Dillinger-Xx** [comment]
If it matters, I never payed any attention to the price. I've never bought an Arrow. I do, however like @Hecter94 's explanation as why it costs so much. It just adds some flavor to the game. It's exactly the same reason luxury cars cost more. Luxury features doesn't make them any more reliable.

**Comment by brendanjones** [comment]
Righto thanks for the answers everyone, and I’ll keep my hands off the Arrow and its silly price tag ;)
